.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,78 @@
+name: aws-deploy
+
+on:
+  pull_request:
+    branches: '*'
+  push:
+    branches: [main, develop]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  sceptre-deploy:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    needs: pre-commit
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Cache Pipenv virtualenv
+        uses: actions/cache@v2
+        id: pipenv-cache
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-v2-${{ hashFiles('**/Pipfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pipenv-v2-
+      - name: Install Dependencies
+        uses: VaultVulp/action-pipenv@v2.0.1
+        if: steps.pipenv-cache.outputs.cache-hit != 'true'
+        env:
+          PIPENV_NOSPIN: 'true'
+          WORKON_HOME: ~/.local/share/virtualenvs
+          PIPENV_CACHE_DIR: ~/.local/share/pipcache
+        with:
+          command: install --dev
+      # Deployment to Development/Sandbox AWS Account
+      - name: Assume AWS Role (Development Account)
+        uses: aws-actions/configure-aws-credentials@v1
+        if: github.ref == 'refs/heads/develop'
+        with:
+          aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.CI_SERVICE_USER_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::035458030717:role/SceptreBuildAccessRole
+          role-duration-seconds: 1200
+      - name: Deploy 'dev' Stack Group to AWS Dev Account
+        uses: VaultVulp/action-pipenv@v2.0.1
+        if: github.ref == 'refs/heads/develop'
+        with:
+          command: run sceptre launch dev --yes
+
+      # Commented out until the AWS account becomes available. Track this PR:
+      # https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/135
+
+      # Deployment to Production AWS Account
+      # - name: Assume AWS Role (Production Account)
+      #   uses: aws-actions/configure-aws-credentials@v1
+      #   if: github.ref == 'refs/heads/main'
+      #   with:
+      #     aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
```

Comment:
you'll probably need to setup a separate set of ENV Vars for prod account

Reply:
I took this pattern from the `organizations-infra` [repo](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/aws-deploy.yaml), where the same service user assumes different roles for deploying to each account. 

@zaro0508: Is there an advantage with using one service user with different roles per account vs different service users per account?